neo/tools/debugger/DebuggerClient.cpp

```diff
@@ -187,19 +213,27 @@ Handle the DBMSG_BREAK message send from the server.  This message is handled
 by caching the file and linenumber where the break occured.
 ================
 */
-void rvDebuggerClient::HandleBreak ( msg_t* msg )
+void rvDebuggerClient::HandleBreak ( idBitMsg* msg )
 {
 	char filename[MAX_PATH];
 
 	mBreak = true;
 
 	// Line number
-	mBreakLineNumber = MSG_ReadInt ( msg );
+	mBreakLineNumber = msg->ReadInt ( );
 
 	// Filename
-	MSG_ReadString ( msg, filename, MAX_PATH );
+	msg->ReadString ( filename, MAX_PATH );
 	mBreakFilename   = filename;
 
+#if D3_SIZEOFPTR == 4 // 32 bit
+	int ptr32b = msg->ReadInt();
+	mBreakProgram = (idProgram*)ptr32b;
+#else
+	int64_t ptr64b = msg->ReadInt64();
```

Comment:
I wonder if
1. this program pointer sent around is even useful and used
2. we should always send an int64 so the network protocol is the same on 32bit and 64bit

Reply:
I have changed to always use the 64_t. that will be that.
But, i might be able to remove the usage/sharing of this pointer all together. I have to change some functions to resolve the target program from a file name, i believe that is how its done originally.

Also, this actually forces the client and server  to run on the same machine to be able to debug.. where it would be nice to be able to debug crossplatform between machines.